test/common_utils.py

```diff
@@ -35,7 +36,7 @@ def loop(op, in_dims, out_dim, batch_size, *batched_args, **kwarg_values):
     return loop_out
 
 
-def get_exhaustive_batched_inputs(arg_values, kwarg_values, batch_size=3):
+def get_exhaustive_batched_inputs(arg_values, kwarg_values, batch_size=3, first_dim_only=False):
```

Comment:
nit: this reads better as `get_exhaustive_batched_inputs(arg_values, kwarg_values, batch_size=3, bdims=(0,  -1))`. This function however only supports `bdims=(0, -1)` and `bdims=(0,)`. If you agree that explicitly writing out the bdims is better for readability then we want to assert that bdims is either `(0,)` or `(0, -1)` to avoid needing to rewrite the function now